azkaban-common/src/main/java/azkaban/trigger/Trigger.java

```diff
@@ -120,47 +120,6 @@ public void setContext(Map<String, Object> context) {
     this.context = context;
   }
 
-  public Trigger(long lastModifyTime, long submitTime, String submitUser,
```

Comment:
Nice clean up.

Would it be better to keep the clean up code in a separate pull request?